shell/platform/glfw/flutter_glfw.cc

```diff
@@ -65,6 +65,10 @@ struct FlutterDesktopWindowControllerState {
   // has been added since it was last removed).
   bool pointer_currently_added = false;
 
+  bool pointer_currently_down = false;
```

Comment:
Please comment these fields, as with the other fields.

Reply:
Updated.